Update readme

thokkat opened the pull request:
Update readme to better reflect the current state of OpenGothic. I linked to some of your explanations regarding mod support.

Try commented on README.md:
```diff
@@ -8,26 +8,75 @@ The goal of this project is to make a feature complete Gothic game client compat
+1. Download a [Pre-Release](https://github.com/Try/opengothic/releases/latest) or build from [CI](https://ci.appveyor.com/project/Try/opengothic/history) and extract into a folder of your choice.
+2. Open `Gothic2Notr.bat`:
+
+   `Gothic2Notr.exe -g "C:\Program Files (x86)\Steam\steamapps\common\Gothic II"`   
```
Hm, should we mention, that on windows OpenGothic can detect some common install paths?

Try replied:
Also, on mac game files are expected to be in `~/Library/Application Support/OpenGothic` by default - this is common for MacSourcePorts projects

thokkat replied:
> should we mention, that on windows OpenGothic can detect some common install paths?

I wasn't sure, is auto detect used even if a wrong path is provided via `-g`? Then we could tell user to just run bat and change only if it fails. 
Can exe on windows be started by double clicking?  Didn't want to bother average user with command line.

I change mac section, also forgot about AUR for Arch.

Try replied:
> I wasn't sure, is auto detect used even if a wrong path is provided via -g?

No, detect works only if path is not specified

Try commented:
Hi, @thokkat thanks for updated readme!

I left few comments on review

thokkat commented on README.md:
```diff
@@ -8,26 +8,75 @@ The goal of this project is to make a feature complete Gothic game client compat
+Supported systems and graphic api's are Windows (DX12, Vulkan), Linux (Vulkan) and MacOS (Metal).
+
+#### Performance
+OpenGothic is known to run poorly on Intel integrated graphics and is barely playable (<5 Fps) on Low-End devices such as Raspberry-Pi. Mainstream graphics cards like a GTX 1060 can run the game mostly at 100+ Fps in FullHD. Perfomance related ingame menu options are `Cloud Shadows` (SSAO, off by default) and `Radial Fog` (on). Raytracing is enabled by default if capable hardware is detected. To  ...
```
>It'd 40-fps on my Intel! :D

Ok I got ~20 :D. I adjust it and the options.

>to separated section.

Actually I think best would be to move this to an faq page on wiki we link to where everything which is not directly related to running/building/status of project can be looked up. Also common questions like how to limit fps etc. I could write something.

Try replied:
OK, lets not dive too deep on FPS topic (also it will become obsolete quite quickly)
Let's just enlist graphics options.
Also worth mention that drawing distance is ignored :)

thokkat commented:
Addressed the changes and made some rephrasing/reordering.

Moved Raspberry pi mention to work in progress. 
Removed performance numbers. Removed default labels (can be viewed ingame) except for command line.
Added dedicated MacOS section for playing.
Added AUR build for Arch. Mentioned wine as last resort
Added recommended label for Winsows Pre-Release to give user some guidance what to choose. Wanted to do the same for Mac Source Ports but version is outdated so I let it as is.

Nindaleth commented on README.md:
```diff
@@ -8,26 +8,96 @@ The goal of this project is to make a feature complete Gothic game client compat
+## How to play
+### Windows
+1. If not already done install Gothic. OpenGothic comes with auto-path detection if your Gothic files are in a common path.
+    * "C:\Program Files (x86)\JoWooD\Gothic II"
```
`installdetect.cpp` seems to check only for steamapps and Akella Games (also not entirely sure which Program Files path is actually checked by OpenGothic on 64-bit systems)

as an aside beyond the scope of this PR, checking for default GOG install path would be nice too

Try commented on README.md:
```diff
@@ -95,7 +164,7 @@ cmake --build ./build --target Gothic2Notr
 | `-save q`              | load the quick save on start                                     |
 | `-save <number>`       | load a specified save-game slot on start                         |
-| `-v -validation`       | enable Vulkan validation mode                                    |
+| `-v -validation`       | enable validation mode                                           |
```
without mention of Vulkan it's not clear what sort of validation is it.
Maybe "enable validation layers for graphics api"

Try commented on README.md:
```diff
@@ -8,26 +8,96 @@ The goal of this project is to make a feature complete Gothic game client compat
 
-The original game has been replicated fully; you can complete both the main quest and the addon.
-Check out the [bugtracker](https://github.com/Try/OpenGothic/issues) for a list of known issues.
+The original game has been replicated fully; you can complete both the main quest and the addon. OpenGothic runs reasonably well except on Low-End devices such as Raspberry-Pi. Check out the [bugtracker](https://github.com/Try/OpenGothic/issues) for a list of known issues.
```
Not like we care about Rasbery-PI...
Probably better rephrase it somehow to indicate that build is aiming for mid-high end gpus, preferably with mesh-shader/rt-support

Try commented on README.md:
```diff
@@ -8,26 +8,96 @@ The goal of this project is to make a feature complete Gothic game client compat
+
+   `exec "$DIR/Gothic2Notr" "$@" -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"`
+
+4. Run `Gothic2Notr.sh`
```
Probably it's easier to simply 'Run `Gothic2Notr.sh` -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II" '

So step 3 can be about running with `-g` and step 4 can mention that it's possible to edit script

thokkat replied:
Exe can't find Tempest library because export is missing. I guess then running script is better than export+command?

Try replied:
no sure what do you mean by Exe. script should take care of LD_LIBRARY_PATH and forward application arguments to opengothic

thokkat replied:
Damn it sorry was overlooking the .sh .

thokkat replied:
Ok changed it :)

Try commented on README.md:
```diff
@@ -61,32 +131,31 @@ make -C build -j $(nproc)
+## Available Graphic options
+
+*  SSAO (mapped to `Cloud Shadows`)
+*  Radial Fog
```
* Volumetric fog (mapped to `Radial fog`)
* Reflections

thokkat commented:
Addressed requested changes (except one, see comment above).

Try commented:
Cool, merged!
